### PR TITLE
Ginormous fixees and updates.

### PR DESCRIPTION
**Pull request by JakeDaBoss**
@NanakoAC @Snypehunter007 @WestHybrid your reviews are needed. If you need less code, just look at the changelog.

### CODE REVIEW

File: code/modules/item_worth/worths_list.dm [hunk 1]
```diff
@@ -25,21 +25,17 @@ var/list/worths = list(
...
-					/obj/item/weapon/gun/energy/lasercannon = 4100,
-					/obj/item/weapon/gun/energy/xray = 2900,
-					/obj/item/weapon/gun/energy/sniperrifle = 5400,
+					/obj/item/weapon/gun/projectile/linecutter = 70,
+					/obj/item/weapon/gun/projectile/seeker = 60,
+					/obj/item/weapon/gun/projectile/ripper = 60,
+					/obj/item/weapon/gun/energy/forcegun = 60,
+					/obj/item/weapon/gun/energy/contact = 60,
+					/obj/item/weapon/gun/projectile/divet = 40,
+					/obj/item/weapon/gun/energy/cutter = 40,
```
Comment (ErwinSnowden) [inline code/modules/item_worth/worths_list.dm:34]: I'm fairly certain that this does not do what you anticipate it does, as it the regular energy gun was defined as 3000 and I can't see these being used anywhere in the cargo console code, rather it seems these were used to define prices for traders (I think vox) on bay judging by the blame history. Has this been tested? Cargo on bay has always been very rudimentary and hasn't taken things other than phoron (and apparently mining scan disks) for ages.

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 2]
```diff
@@ -157,10 +157,10 @@
...
 	result_amount = 4
 
-/datum/chemical_reaction/pacid
-	name = "Polytrinic acid"
-	result = /datum/reagent/acid/polyacid
-	required_reagents = list(/datum/reagent/acid = 1, /datum/reagent/acid/hydrochloric = 1, /datum/reagent/potassium = 1)
+/datum/chemical_reaction/tacid
+	name = "Triflic acid"
+	result = /datum/reagent/acid/triflicacid
+	required_reagents = list(/datum/reagent/toxin/potassium_chlorophoride = 1, /datum/reagent/acid/hydrochloric = 1, /datum/reagent/lexorin = 1)
```
Comment (ErwinSnowden) [inline code/modules/reagents/Chemistry-Recipes.dm:163]: Renaming the reagent breaks the recipes where it's used, mainly **metal foam and nitroglycerin**, metal foam nades are actually used commonly and I'm pretty sure this will cause issues when metalfoam grenades call for polyacid, if they use actual chemicals in their code.

File: code/modules/research/designs.dm [hunk 3]
```diff
@@ -583,11 +583,26 @@
...
+	build_path = /obj/item/weapon/gun/energy/cutter
+	sort_string = "TAEAB"
+
+/datum/design/item/weapon/plasmacutter
+	name = "211-V plasma cutter"
+	id = "plasmacutter"
+	req_tech = list(TECH_COMBAT = 3, TECH_MATERIAL = 4, TECH_POWER = 3)
+	materials = list(MATERIAL_STEEL = 15000, MATERIAL_GLASS = 5000. "gold" = 400)
+	build_path = /obj/item/weapon/gun/energy/cutter
+	sort_string = "TAEAC"
```
Comment (ErwinSnowden) [inline code/modules/research/designs.dm]:
```suggestion
/datum/design/item/weapon/plasmacutter
	name = "211-V plasma cutter"
	id = "plasmacutter"
	req_tech = list(TECH_COMBAT = 3, TECH_MATERIAL = 4, TECH_POWER = 3)
	materials = list(MATERIAL_STEEL = 15000, MATERIAL_GLASS = 5000. "gold" = 400)
	build_path = /obj/item/weapon/gun/energy/cutter/plasma
	sort_string = "TAEAC"
```
Make sure to check that it actually makes the desired item next time 😄

File: code/modules/reagents/Chemistry-Recipes.dm (hunk 2 above)
Comment (JakeDaBoss) [reply code/modules/reagents/Chemistry-Recipes.dm:163]: Oops. I'll have to fix that.

File: code/modules/item_worth/worths_list.dm (hunk 1 above)
Comment (JakeDaBoss) [reply code/modules/item_worth/worths_list.dm:34]: This is the exports list. I'm fairly sure this is the worths list for exports. You're able to export things out. Haven't tested yet.

File: code/modules/mob/living/carbon/human/species/necromorph/exploder.dm [hunk 4]
```diff
@@ -12,6 +12,7 @@
 	total_health = 85	//It has high health for the sake of making it a bit harder to destroy without targeting the pustule. Exploding the pustule is always an instakill
 	biomass = 70
 	mass = 50
+	species_flags = SPECIES_FLAG_NO_PAIN | SPECIES_FLAG_NO_MINOR_CUT | SPECIES_FLAG_NO_POISON  | SPECIES_FLAG_NO_BLOCK | SPECIES_FLAG_NO_SLIP
```
Comment (NanakoAC) [inline code/modules/mob/living/carbon/human/species/necromorph/exploder.dm]: This is not a fix, it was working as intended. Revert these changes to species flags.

I pay careful attention to the flags on every new necromorph, their existing state was designed, not an accident

**Review by NanakoAC — CHANGES_REQUESTED** [review]

File: code/game/gamemodes/game_mode.dm [hunk 5]
```diff
@@ -346,14 +346,41 @@ var/global/list/additional_antag_types = list()
 			else if(isghost(M))
 				ghosts++
 
+		if(M.get_preference_value(/datum/client_preference/play_admin_midis) == GLOB.PREF_YES)
+			if (escaped_total < 1)
```
Comment (NanakoAC) [inline code/game/gamemodes/game_mode.dm:350]: these values need to be percentages of historic crew count, that number is already tracked in the code that calculates when enough people have died to trigger an auto ending

File: code/game/movietitles.dm [hunk 6]
```diff
@@ -96,13 +93,13 @@ client
...
-	possible_titles += "THE CREW LEARNS ABOUT [pick("LOVE", "DRUGS", "THE DANGERS OF MONEY LAUNDERING", "XENIC SENSITIVITY", "INVESTMENT FRAUD", "KELOTANE ABUSE", "RADIATION PROTECTION", "SACRED GEOMETRY", "STRING THEORY", "ABSTRACT MATHEMATICS", "[pick("TAJARAN", "UNATHI", "SKRELLIAN", "DIONAN", "KHAARMANI", "VOX", "SERPENTID")] MATING RITUALS", "ANCIENT CHINESE MEDICINE")]"
-	possible_titles += "A VERY [pick("NANOTRASEN", "EXPEDITIONARY", "DIONA", MATERIAL_PHORON, "MARTIAN")] CHRISTMAS"
-	possible_titles += "[pick("GUNS, GUNS EVERYWHERE", "THE LITTLEST ARMALIS", "WHAT HAPPENS WHEN YOU MIX MAINTENANCE DRONES AND COMMERCIAL-GRADE PACKING FOAM", "ATTACK! ATTACK! ATTACK!", "SEX BOMB")]"
+	possible_titles += "THE [pick("DOWNFALL OF", "RISE OF", "TROUBLE WITH", "FINAL STAND OF", "DARK SIDE OF")] [pick("THE MEDICAL BAY", "INCOMPETENCE", "SHITCURITY", "RANDOM ARRESTS", "MURDER", "COMMAND", "CARGO VS SECURITY", "[uppertext(GLOB.using_map.station_name)]")]"
```
Comment (NanakoAC) [inline code/game/movietitles.dm]: shitcurity is memes, please don't have that there

humanity, dignity and sanity were good

File: code/game/movietitles.dm [hunk 7]
```diff
@@ -96,13 +93,13 @@ client
...
+	possible_titles += "THE CREW GETS [pick("SLASHERED", "PUKED ON", "AN INCURABLE DISEASE", "CORPSED", "A VALUABLE HISTORY LESSON", "A BREAK", "HIGH", "TO LIVE", "TO RELIVE THEIR NIGHTMARES", "EMBROILED IN WAR ON THE NECRONS", "SERIOUS ABOUT [pick("DRUG ABUSE", "CRIME", "PRODUCTIVITY", "ANCIENT AMERICAN CARTOONS", "SPACEBALL")]")]"
```
Comment (NanakoAC) [inline code/game/movietitles.dm]: necrons are a whole different franchise, its not a short form of necromorphs

File: code/game/objects/items/weapons/storage/toolbox.dm [hunk 8]
```diff
@@ -8,11 +8,11 @@
 	icon_state = "red"
 	item_state = "toolbox_red"
 	obj_flags = OBJ_FLAG_CONDUCTIBLE
-	force = 20
-	attack_cooldown = 21
+	force = 12
```
Comment (NanakoAC) [inline code/game/objects/items/weapons/storage/toolbox.dm]: use one of the defines for this in _defines/weapons.dm

File: code/modules/item_worth/materials.dm [hunk 9]
```diff
@@ -2,64 +2,64 @@
 	var/value = 1
 
 /material/uranium
-	value = 100
+	value = 75
```
Comment (NanakoAC) [inline code/modules/item_worth/materials.dm:5]: Did you check that this actually works like we think it does? someone raised doubts

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm [hunk 10]
```diff
@@ -80,7 +80,7 @@
 /datum/reagent/dylovene
 	name = "Dylovene"
 	description = "Dylovene is a broad-spectrum antitoxin used to neutralize poisons before they can do significant harm."
-	taste_description = "a roll of gauze"
+	taste_description = "a groll of gauze"
```
Comment (NanakoAC) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm]: ???

the description was silly before, but i'm not sure this is much better

File: code/modules/reagents/Chemistry-Reagents/acid.dm [hunk 11]
```diff
@@ -83,14 +83,14 @@
...
+/datum/reagent/acid/triflicacid
+	name = "Triflic acid"
+	description = "Triflic acid is a an extremely corrosive chemical substance."
 	taste_description = "acid"
 	reagent_state = LIQUID
-	color = "#8e18a9"
-	power = 5
-	meltdose = 4
+	color = "#939c38"
+	power = NECROMORPH_ACID_POWER + 0.5
```
Comment (NanakoAC) [inline code/modules/reagents/Chemistry-Reagents/acid.dm]: use multipliers instead of additions when balancing relative to a define. 

NECROMORPH_ACID_POWER * 1.5 or something like that, or perhaps make it its own define

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 12]
```diff
@@ -449,7 +449,7 @@
 /datum/chemical_reaction/plastication
 	name = "Plastic"
 	result = null
-	required_reagents = list(/datum/reagent/acid/polyacid = 1, /datum/reagent/toxin/plasticide = 2)
+	required_reagents = list(/datum/reagent/acid/hydrochloric = 1, /datum/reagent/toxin/plasticide = 2)
```
Comment (NanakoAC) [inline code/modules/reagents/Chemistry-Recipes.dm:452]: why are things requiring hydrochloric acid instead of triflic acid? the latter is what you've replaced polyacid with. This seems to make a lot of recipes wayyy easier to make

File: code/modules/research/designs.dm [hunk 13]
```diff
@@ -583,11 +583,26 @@
 /datum/design/item/weapon/contactbeam
 	name = "C99 Supercollider Contact Beam"
 	id = "contactbeam"
-	req_tech = list(TECH_COMBAT = 3, TECH_MATERIAL = 5, TECH_POWER = 3)
-	materials = list(MATERIAL_STEEL = 5000, MATERIAL_GLASS = 1000, "uranium" = 500)
+	req_tech = list(TECH_COMBAT = 5, TECH_MATERIAL = 5, TECH_POWER = 3)
+	materials = list(MATERIAL_STEEL = 5000, MATERIAL_GLASS = 1000, "diamond" = 400)
```
Comment (NanakoAC) [inline code/modules/research/designs.dm:587]: is diamond reasonably attainable? I put this here originally just taking the place (and costs) of the advanced energy gun, since people were making those

contact beam isn't meant to be especially rare if science wants to manufacture them

**Review by NanakoAC — CHANGES_REQUESTED** [review]
some things need a bit of tweaking

File: code/modules/research/designs.dm (hunk 13 above)
Comment (ErwinSnowden) [reply code/modules/research/designs.dm:587]: Diamond happens to appear on most rounds, however raising the combat tech is just an unneeded, superfluous change. I'm not sure why it had to be made.

**Comment by ErwinSnowden** [comment]
I am not fond of the changes to the end title screen (and I'm sure others wont be either), it's very iconic and thunderdome is a very upbeat song that lets me know when the round is ended, and I'm sure it does to others which tells us to ready up for the next round.

File: code/modules/hydroponics/seed_datums.dm [hunk 14]
```diff
@@ -212,7 +212,6 @@
 	name = "bluetomato"
 	seed_name = "blue tomato"
 	display_name = "blue tomato plant"
-	mutants = list("bluespacetomato")
 	chems = list(/datum/reagent/nutriment = list(1,20), /datum/reagent/lube = list(1,5))
 
 /datum/seed/tomato/blue/New()
```
Comment (ErwinSnowden) [inline code/modules/hydroponics/seed_datums.dm:217]: The trait can still be randomly mutated, and bluespace tomatoes themselves are un-alluring and rarely seen anyway.

File: code/game/objects/items/weapons/storage/toolbox.dm [hunk 15]
```diff
@@ -8,11 +8,11 @@
...
+	attack_cooldown = 23
 	melee_accuracy_bonus = -20
-	throwforce = 10
-	
+	throwforce = 14
```
Comment (ErwinSnowden) [inline code/game/objects/items/weapons/storage/toolbox.dm]: Why is this needed? People are just going to pick baseball bats instead to do their B&Es then, and metal can commonly be found in lockers. Why shouldn't an iron toolbox do rather well as a reserve weapon? As it isn't sharp, it poses little-to-no threat to necromorphs.

Comment (NanakoAC) [reply code/game/objects/items/weapons/storage/toolbox.dm]: 20 was a pretty crazy amount of force, most of the good weapons in our code have about 15 damage, higher values are bay holdovers we haven't gotten around to rebalancing.

A toolbox is a heavy object but not really intended as a weapon, its shape isn't amenable for efficient swinging. A baseball bat would make more sense

File: code/modules/research/designs.dm (hunk 13 above)
Comment (JakeDaBoss) [reply code/modules/research/designs.dm:587]: @NanakoAC

Diamond became decently attainable now that the drill heads are fixed and can be used.

File: code/modules/reagents/Chemistry-Recipes.dm (hunk 12 above)
Comment (JakeDaBoss) [reply code/modules/reagents/Chemistry-Recipes.dm:452]: Correct. Triflic Acid received major nerfs. Keep in mind that Triflic / Former Poly now requires a chemistry reagent you need Phoron for to make.

This meant I had to do a head scratch and consider in-depth whether existing stuff should use Triflic or just Hydrochloric. It became the latter, to make the Chemistry change lesser.

File: code/modules/reagents/Chemistry-Reagents/acid.dm (hunk 11 above)
Comment (JakeDaBoss) [reply code/modules/reagents/Chemistry-Reagents/acid.dm]: Will do the multiplier.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm (hunk 10 above)
Comment (JakeDaBoss) [reply code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm]: I'm not sure how this made it in, actually. I'm 100% certain this is a typo I made during Reagent changes.

File: code/modules/hydroponics/seed_datums.dm (hunk 14 above)
Comment (JakeDaBoss) [reply code/modules/hydroponics/seed_datums.dm:217]: It doesn't.

File: code/game/gamemodes/game_mode.dm (hunk 5 above)
Comment (JakeDaBoss) [reply code/game/gamemodes/game_mode.dm:350]: West wanted it this way, actually.

**Review by NanakoAC — APPROVED** [review]
